src/spid_sp_test/responses/response_mods.py

```diff
@@ -7,15 +7,25 @@
 
 
 def dynamic_acr(check: SpidSpResponseCheck, response_obj: SpidSpResponse, **kwargs):
-    if check.get_acr() != check.response_attrs["AuthnContextClassRef"]:
```

Comment:
I'd suggest you to leave this if as it was, because it's a concrete requirement, otherwise you'll do a memory initialization with the try/except for nothing

Reply:
Correct me if I'm wrong: if you leave this, you'll make the following checks effective ONLY if the two levels (requested vs obtained) are different. However, this does not allow to correctly implement "better" comparison value.